src/App.js

```diff
@@ -1,13 +1,69 @@
-import React from 'react';
-import { BrowserRouter as Router, Routes, Route } from 'react-router-dom';
-import Greeting from './components/Greeting';
-
-const App = () => (
-  <Router>
-    <Routes>
-      <Route exact path="/" element={<Greeting />} />
-    </Routes>
-  </Router>
-);
+import React, { useEffect, useState } from 'react';
+import { BrowserRouter as Router, Route, Routes } from 'react-router-dom';
+import { useDispatch } from 'react-redux';
+import SplashPage from './components/SplashPage';
+import HomePage from './components/HomePage';
+import LoginForm from './components/sessions/LoginForm';
+import RegistrationForm from './components/sessions/RegistrationForm';
+import HouseForm from './components/addHouse';
+import ReservationsList from './components/ReservationsList';
+import ReservationForm from './components/ReservationForm';
+import { fetchHouses } from './redux/houses/housesSlice';
+import NavBar from './components/NavBar';
+import HouseDetails from './components/HouseDetails';
+import DeleteHouse from './components/DeleteHouse';
+
+const App = () => {
```

Comment:
- [ ] Kindly, remove the old commented code found in:

1. NavBar.js
2. ReservationForm.jsx